modules/ROOT/pages/tuning-repeatable-nonrepeatable.adoc

```diff
@@ -0,0 +1,84 @@
+= Repeatable vs. Non-Repeatable Streaming
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Understanding streaming strategies and properly establishing which one applies best depending on your use case, is key to get the most of your flows in terms of performance.
+
+To determine the streaming strategy that best suits your use case, first identify if you need:
+
+* A non-repeatable stream (read payload only once) or
+* A repeatable stream (read payload more than once)
+
+== Non-Repeatable Streaming
```

Comment:
```suggestion
== When to Use Nonrepeatable Streaming
```
But I wonder if we really need this topic, since we have have defined this already and don't have more to say.
Next time you look at this, consider omitting these definition topics and promoting the subtopics about Repeatable Streaming:

"==When to Use Repeatable vs. Nonrepeatable Streaming
(improves flow
two choices: repeat and nonrepeat
nonrepeat = one time
repeat = more than once (do we really have to define "repeat" and nonrepeat? they are common terms) AND two types, file store and in-memory)

===When to Use a File-Store Repeatable Stream
(include buffer sizing to do so)

===When to Use an In-Memory Repeatable Stream
(example and buffer sizing)

Reply:
Copyedits below: